presto-hive/src/test/java/com/facebook/presto/hive/TestHiveSplitManager.java

```diff
@@ -459,24 +459,13 @@ private void assertRedundantColumnDomains(Range predicateRange, PartitionStatist
 
         // Prepare partition with stats
         PartitionWithStatistics partitionWithStatistics = new PartitionWithStatistics(
-                new Partition(
-                        "test_db",
-                        "test_table",
-                        ImmutableList.of(PARTITION_VALUE),
-                        new Storage(
-                                fromHiveStorageFormat(ORC),
-                                "location",
-                                Optional.empty(),
-                                true,
-                                ImmutableMap.of(),
-                                ImmutableMap.of()),
-                        COLUMNS,
-                        ImmutableMap.of(),
-                        Optional.empty(),
-                        false,
-                        true,
-                        0,
-                        0),
+                new Partition("test_db", "test_table", ImmutableList.of(PARTITION_VALUE), new Storage(
```

Comment:
Can you check if these test changes follow our guidelines on line width, and if so, could you please revert to the previous formatting? https://github.com/prestodb/presto/blob/master/CONTRIBUTING.md#code-style

180 characters is a general guideline, but when there's lots of arguments, one argument per line is more readable.

Reply:
I did an IntelliJ reformat on this fragment. It moved some lines a few spaces left, otherwise left them unchanged. It's useful to stick with what the autoformatter does to avoid a lot of churn. 

Ideally we should autoformat unconditionally without human intervention, but that's going to take a little time.

Reply:
Definitely makes sense, but we do note to be careful about using the auto-formatting tool in the contributing guide, and we also lose history of the individual lines for these tests, so best to revert to the previous formatting.

Reply:
often we'll put reformatting in a separate commit so it doesn't clutter up the real changes, but I disagree about leaving it behind entirely. if the new formatting is correct, I think it's good to commit it so people don't need to keep undoing autoreformats manually. I wouldn't worry about losing history (you can just go back farther in the git commit history if you need to).

Reply:
anyway, undone for now to unblock this

Reply:
The trouble is whatever is auto-formatted by IntelliJ isn't necessarily guaranteed to be correct, which is the case here because we exceed line width guidelines linked above.  I also find the formatting confusing: `TestParquetQuickStatsBuilder` puts the remaining arguments after the `new Storage` argument on the same line, yet for some reason we put the arguments for `new Storage` all on individual lines.  I find this mix match of formatting confusing and think we should just fix it.

I also agree than in principle it would be great to format things consistently and automatically, and in such a way that it perfectly guaranteed alignment with our style guide.

Reply:
There are several existing line length violations in this file, none of which seem to be on lines I touched. I don't see any line length violations in TestHIveSplitManager on lines I touched.

Yes, the formatting is confusing. That's caused by inlining way too many, too complex method arguments. Using more local variables would clarify this greatly. But this problem is not new with this PR, and shouldn't be blocking.

Reply:
To be clear, I'm just referring to `TestParquetQuickStatsBuilder` and `TestQuickStatsProvider`, the rest I think can remain (although ideally these should be in a separate reformatting commit).